.env

```diff
@@ -0,0 +1 @@
+HOST=0.0.0.0
```

Comment:
:question:

Reply:
Oops. I commited everything. That's only needed for me, I'll remove